src/styles/main.scss

```diff
@@ -16,18 +16,18 @@
 @import "Question";
 @import "StudentQuestionContainer";
 
-body {background: grey; font-family: $font}
+body {
+  background: #d4d6d9;
+  font-family: $font
+}
 
 .main-container {
   color: white;
-  position: fixed;
-  top: $nav-height; bottom: 0;
-  width: 100vw;
+  width: 97vw;
+  height: 100vh;
 }
 
 .side-margins {
-  position: absolute;
-  top: 0; bottom: 0;
-  left: 20%; right: 20%;
+   padding: 0% 15% 2% 17%;
```

Comment:
Also why?

Reply:
We don't want to utilize position absolute unless necessary. The container is positioning the cards in the same manner with the one padding rule.

Reply:
Might want to use $nav-height in there somewhere in case that changes.